lib/pact.ex

```diff
@@ -43,11 +43,18 @@ defmodule Pact do
   end
 
   @doc "Override all calls to `name` in `pid` with `module`"
-  def override(pid, name, module) do
+  def override(pid, name, module) when is_atom(module) do
     name = to_string(name)
     GenServer.cast(__MODULE__, {:override, pid, name, module})
   end
 
+  def override(pid, name, overridden_functions) when is_list(overridden_functions) do
+    name = to_string(name)
+    {:module, fake_module_name, _, _} = create_fake_module(Pact.Fake)
```

Comment:
I need to figure out a way to dynamically create a module name here so that there are no collisions